pkg/querier/remote_read.go

```diff
@@ -141,7 +143,13 @@ func remoteReadStreamedXORChunks(
 
 	for i, qr := range req.Queries {
 		if err := processReadStreamedQueryRequest(ctx, i, qr, q, w, f, maxBytesInFrame); err != nil {
-			level.Error(logger).Log("msg", "error streaming remote read response", "err", err)
+			if errors.Is(err, context.Canceled) {
+				level.Warn(logger).Log("msg", "the remote read request has been canceled", "err", err)
```

Comment:
IMO this isn't worth logging here.  Or perhaps on an `info` level at most. It adds noise for something unactionable. Plus, we already have the metrics by response code, so we can spot if 499 become very common (if we care about it at some point)

Reply:
Ok fair enough. I remove the log.

Reply:
Done in https://github.com/grafana/mimir/pull/6934/commits/0c0b8175ac6fdf786fd936a9aba25f85e45dddbd. I've added a comment to explain why we don't log specifically for this case.